08LessonEight/in-class-project/exercise-08.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Recent</title>
+  <link rel="stylesheet" href="css/exercise-style.css" />
+</head>
+<body>
+  <header>
+    <h1>Recent from Blog</h1>
+    <p>Here's the latest and greatest thoughts from our contributors. Select from dozens of articles, loaded with industry insight and creative tips.</p>
+  </header>
+  <section class="clear-fix">
+    <div class="clear-fix">
+      <img id="keyboard" src="img/keyboard.jpg" />
```

Comment:
You are correct.  It's weird because the img file name worked on live server, but not on hosted github.  Fixed img file name.